lib/sqlalchemy/dialects/mysql/json.py

```diff
@@ -13,8 +13,7 @@
 class JSON(sqltypes.JSON):
     """MySQL JSON type.
 
-    MySQL supports JSON as of version 5.7.  Note that MariaDB does **not**
-    support JSON at the time of this writing.
+    MySQL supports JSON as of version 5.7.  MariaDB support JSON (as alias for LONGTEXT) from version 10.2
```

Comment:
We have a max line length of 80char. Could you please reformat this line?

Reply:
needs grammatical correction as well.

Reply:
@zzzeek could you please suggest your variant?

Reply:
@CaselIT done

Reply:
I think he was talking about `support`->`supports` like in the MySQL version

Reply:
"MariaDB supports JSON (as an alias for LONGTEXT) as of version 10.2."

Reply:
@zzzeek , my bad :)

All are fixed. Thanks